packages/e2e-tests/specs/writing-flow.test.js

```diff
@@ -55,14 +55,15 @@ describe( 'adding blocks', () => {
 		// Arrow up into nested context focuses last text input
 		await page.keyboard.press( 'ArrowUp' );
 		activeElementText = await page.evaluate( () => document.activeElement.textContent );
-		expect( activeElementText ).toBe( 'Second col' );
+		expect( activeElementText ).toBe( '2nd col' );
 
 		// Arrow up in inner blocks should navigate through (1) column wrapper,
 		// (2) text fields.
 		await page.keyboard.press( 'ArrowUp' );
+		await page.waitForSelector( '.is-selected[data-type="core/column"]:focus' );
```

Comment:
If I understand https://github.com/WordPress/gutenberg/pull/14961/files#r305412715 correctly, this is no longer necessary? Could it be removed?

Reply:
Yes, I believe that's correct. Removed in 7a0a201.